apps/system/js/utility_tray.js

```diff
@@ -250,22 +248,12 @@ window.UtilityTray = {
           return;
         }
 
-        // Prevent swipe down gesture when already opened/opening.
-        if (target !== this.grippy && this.showing) {
-          return;
-        }
-
-        if (target !== this.overlay && target !== this.grippy &&
-            evt.currentTarget !== this.statusbarIcons &&
-            evt.currentTarget !== this.topPanel) {
-          return;
+        if (this._isDraggable(target)) {
+          this.onTouchStart(evt.touches[0]);
+        } else {
+          evt.stopPropagation();
         }
 
-        if (this._shouldPrevent(target)) {
```

Comment:
I'm not sure if it's still needed. There is no comment explaining why the `evt.preventDefault()` is required :disappointed: